app/src/main/res/values/colors.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="purple_200">#FFBB86FC</color>
-    <color name="purple_500">#FF6200EE</color>
-    <color name="purple_700">#FF3700B3</color>
-    <color name="teal_200">#FF03DAC5</color>
-    <color name="teal_700">#FF018786</color>
+    <color name="primary_light">#FFBB86FC</color>
+    <color name="primary_dark">#FF6200EE</color>
+    <color name="primary_dark_dark">#FF3700B3</color>
+    <color name="secondary_light">#FF03DAC5</color>
+    <color name="secondary_dark">#FF018786</color>
```

Comment:
Refactored color names to express their role in the color scheme, for clarity, and to follow these conventions.
https://codepen.io/nicetransition/pen/QjwbRg